src/cockpit/osinfo.py

```diff
@@ -0,0 +1,25 @@
+# supported OSes for beibooting; entries are os-release keys
+# keep this in sync with bots/lib/testmap.py
+supported_oses: 'list[dict[str, str | None]]' = [
+    # rolling release
+    {"ID": "arch", "VERSION_ID": None},
+
+    # match/describe CentOS separately, it's the upstream of all RHEL clones
+    {"ID": "centos", "PLATFORM_ID": "platform:el9"},
+    {"ID": "centos", "PLATFORM_ID": "platform:el10"},
+    {"PLATFORM_ID": "platform:el8"},
```

Comment:
Allowing RHEL 8 means I can beiboot into a machine and get files there which won't work so maybe we should not allow that?

Also notably opensuse-tumbleweed is absent but we don't test it yet in CI I suppose :)

Reply:
If that's true, the same would already happen with the Client or the ws container, right? Please remind me why it doesn't work, something something old Python version, or kernel not supporting something fancy that we do in the bridge (fsinfo channel)?

I tried to log into a rhel-8-10 VM with our Flatpak, and it does *not* show Files. I'm not sure, perhaps the "requires cockpit > 318" in the manifest? (But that'd be weird, it's 331). I'll check later.

I'm happy to remove el8 from this list for the time being.

Reply:
We need fsinfo which is not in RHEL 8. Ah right if this is already handled in the manifest that is fine then?

Reply:
Argh, sorry, that was PEBCAK -- my VM already had cockpit-brige.rpm installed. After removing it, the File page appears in the Client, and works quite well.

Reply:
So, in summary: Files in particular works fine. If there is a page which doesn't work somewhere, we need to hide it via manifest conditions, so that it DTRT in ws and Client.